src/content/reference/feature-flags.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Feature Flags
+description: XXX
+sidebar_label: Feature Flags
+id: feature-flags
+---
+
+import Image from "@theme/Image";
+
+Okteto provides a list of variables that serve as feature flags for some of the features we release in our product. Below is a list of those variables that you can use to leverage these newer and experimental features:
+
```

Comment:
It would be good to format the below section as a table to match the information presentation in https://www.okteto.com/docs/reference/okteto-cli/#options

Reply:
Good shout @codyjlandstrom 👍 

<img width="1742" alt="Screenshot 2024-05-10 at 23 04 35" src="https://github.com/okteto/docs/assets/2318450/0c60bf8a-c8c0-47f2-8be4-0942f992cfa3">